entry_types/scrolled/package/src/editor/views/EditSectionTransitionEffectView.module.css

```diff
@@ -97,71 +91,42 @@
     position: absolute;
 }
 
-/*fade Upper Section*/
+/*Reveal, fade Upper Section, & Before After Upper Section*/
+.reveal .upper_section,
+.beforeAfter .upper_section,
 .fade .upper_section {
-    transition: all 2s ease-in-out;
-    position: absolute;
-    z-index: 1;
-}
-
-/*Reveal Upper Section*/
-.reveal .upper_section {
     position: absolute;
     z-index: 1;
 }
 
-/*Before After Upper Section*/
```

Comment:
All of this was not needed, and were using the same properties. Moved transition to :hover above where they now use keyframe transitions.